shtdlib.sh

```diff
@@ -1654,6 +1654,17 @@ function ln_sf {
     debug 10 "Successfully created symlink"
 }
 
+# Create string of random characters
+#  - First param is length, default: 20
+#  - Second param is characters, default: A-Za-z0-9_
+gen_rand_chars() {
+    local length="${1:-20}"
+    local chars="${2:-A-Za-z0-9_}"
+    debug 10 "Creating a string of random characters of length ${length}"
+    LC_CTYPE=C tr -dc ${chars} < /dev/urandom | head -c ${length}
+}
+
```

Comment:
Should probably just be one line

Reply:
Yeah, I thought about it but having it like this gives us the `debug` message which could come in handy with verbose logging